packages/vscode-extension/tests/webviewUtils.ts

```diff
@@ -49,3 +51,33 @@ export async function openFixtureFile(
     options,
   );
 }
+
+export async function selectValue(
+  webview: WebView,
+  elem: string,
+  value: string,
+) {
+  await webview.open();
+  const element = await $(elem);
+  await element.selectByVisibleText(value);
+  await webview.close();
+}
+
+export async function setText(webview: WebView, elem: string, value: string) {
+  await webview.open();
+  await (await $(elem)).clearValue();
+  await webview.close();
+  await webview.open();
+  await (await $(elem)).addValue(value);
+  await webview.close();
```

Comment:
This seems better to use than `elem.setValue()` because that internally clears the value and writes the new one, with the difference that it does not reopen the webview, so the locators can get outdated and the tests would fail.